pom.xml

```diff
@@ -412,7 +412,7 @@
 			<dependency>
 				<groupId>postgresql</groupId>
 				<artifactId>postgresql</artifactId>
-				<version>9.0-801.jdbc4</version>
+				<version>9.4.1208-jdbc42-atlassian-hosted</version>
```

Comment:
Why -atlassian-hosted?

Reply:
that seesm to be how it appears to be from maven central repo 
```
<!-- https://mvnrepository.com/artifact/postgresql/postgresql -->
<dependency>
    <groupId>postgresql</groupId>
    <artifactId>postgresql</artifactId>
    <version>9.4.1208-jdbc42-atlassian-hosted</version>
</dependency>
```